### PR TITLE
ts-migration/valid-expect-in-promise

### PR DESCRIPTION
**Pull request by G-Rath**
This'll probably conflict a little when #376 is merged, but should be fine.

### CODE REVIEW

**Comment by G-Rath** [comment]
Ugh WebStorm gets confused when you switch from js -> ts, b/c it marks the `js` as a generated file, so it told me `getStringValue` wasn't being used, when its needed by `no-large-snapshots` :joy:

File: src/rules/__tests__/valid-expect-in-promise.test.ts [hunk 1]
```diff
@@ -325,12 +196,128 @@ ruleTester.run('valid-expect-in-promise', rule, {
...
     `,
-
     `
       it('name of done param does not matter', (nameDoesNotMatter) => {
         const promise = getPromise();
         promise.then(() => expect(someThing).toEqual(true));
       });
     `,
   ],
+  invalid: [
```
Comment (SimenB) [inline src/rules/__tests__/valid-expect-in-promise.test.ts]: these are just moved right? no actual change?

Comment (G-Rath) [reply src/rules/__tests__/valid-expect-in-promise.test.ts]: Yup - every other test has `valid` at the top, & I think the consistency is nice.

**Comment by G-Rath** [comment]
Want me to handle the conflict?

**Comment by SimenB** [comment]
yes please 🙂

**Comment by SimenB** [comment]
This is great! (too bad git gets so confused by renames :( reviewing locally works way better)

**Pull request merged**